tests/components/UserActions/QueryList.spec.ts

```diff
@@ -269,30 +269,6 @@ describe('QueryList', () => {
             });
         });
 
-        it('should log a search event with "userActionsSubmit" as search cause', () => {
```

Comment:
Why are you removing this test?

Reply:
usageAnalytics is no more mockable by updating the jsui

Reply:
Upgrading from `2.0.342` to `2.8521.7` is a pretty big leap forward indeed. Ok then.